solutions/security/cloud/get-started-with-cspm-for-aws.md

```diff
@@ -21,7 +21,7 @@ This page explains how to get started monitoring the security posture of your cl
 * Minimum privileges vary depending on whether you need to read, write, or manage CSPM data and integrations. Refer to [CSPM privilege requirements](/solutions/security/cloud/cspm-privilege-requirements.md).
 * The CSPM integration is available to all {{ecloud}} users. On-premise deployments require an [Enterprise subscription](https://www.elastic.co/pricing).
 * CSPM only works in the `Default` {{kib}} space. Installing the CSPM integration on a different {{kib}} space will not work.
-* CSPM is supported only on AWS, GCP, and Azure commercial cloud platforms, and AWS GovCloud. Other government cloud platforms are not supported. [Click here to request support](https://github.com/elastic/kibana/issues/new/choose).
+* CSPM supports the AWS, GCP, and Azure commercial cloud platforms, and AWS GovCloud. AWS GovCloud is only supported for agent-based deployments, agentless deployments do not work on this platform. Other government cloud platforms are not supported. [Click here to request support](https://github.com/elastic/kibana/issues/new/choose).
```

Comment:
What's the relationship between CPSM and the platforms listed? Does CPSM support the platforms or do the platforms support CPSM? I ask because this text suggests the former while the text in the other file suggests the latter.